### PR TITLE
zh-translation: /intro/arch_overview/upstream/load_reporting_service.rst

### PR DESCRIPTION
**Pull request by webup**
ref: https://github.com/cloudnativeto/envoy/issues/46

### CODE REVIEW

File: docs/root/intro/arch_overview/upstream/load_reporting_service.rst [hunk 1]
```diff
@@ -1,15 +1,11 @@
 .. _arch_overview_load_reporting_service:
 
-Load Reporting Service (LRS)
+负载报告服务 (LRS)
```
Comment (helight) [inline docs/root/intro/arch_overview/upstream/load_reporting_service.rst]:
```suggestion
负载上报服务 (LRS)
```

File: docs/root/intro/arch_overview/upstream/load_reporting_service.rst [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
 ============================
 
-The Load Reporting Service provides a mechanism by which Envoy can emit Load Reports to a management
-server at a regular cadence.
+负载报告服务提供了一种机制，使 Envoy 可以定期向管理服务器发送负载报告。
```
Comment (helight) [inline docs/root/intro/arch_overview/upstream/load_reporting_service.rst]:
```suggestion
负载上报服务提供了一种机制，使 Envoy 可以定期向管理服务器发送负载报告。
```

File: docs/root/intro/arch_overview/upstream/load_reporting_service.rst [hunk 3]
```diff
@@ -1,15 +1,11 @@
...
 
-This will initiate a bi-directional stream with a management server. Upon connecting, the management
-server can send a :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>`
-to a node it is interested in getting the load reports for. Envoy in this node will start sending
-:ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
-based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
+这将启动一个与管理服务器的双向流。连接后，管理服务器可以发送一个 :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>` 给它有兴趣获取负载报告的节点。这个节点中的 Envoy 将开始发送 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`。这是根据 :ref:`负载报告时间间隔 <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>` 定期进行的。
+
+带负载报告服务的 Envoy 配置可以在 :repo:`/examples/load-reporting-service/service-envoy-w-lrs.yaml` 找到。
```
Comment (helight) [inline docs/root/intro/arch_overview/upstream/load_reporting_service.rst]:
```suggestion
Envoy 负载上报服务的配置可以看这里 :repo:`/examples/load-reporting-service/service-envoy-w-lrs.yaml` 。
```

File: docs/root/intro/arch_overview/upstream/load_reporting_service.rst [hunk 4]
```diff
@@ -1,15 +1,11 @@
...
-The Load Reporting Service provides a mechanism by which Envoy can emit Load Reports to a management
-server at a regular cadence.
+负载报告服务提供了一种机制，使 Envoy 可以定期向管理服务器发送负载报告。
 
-This will initiate a bi-directional stream with a management server. Upon connecting, the management
-server can send a :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>`
-to a node it is interested in getting the load reports for. Envoy in this node will start sending
-:ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
-based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
+这将启动一个与管理服务器的双向流。连接后，管理服务器可以发送一个 :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>` 给它有兴趣获取负载报告的节点。这个节点中的 Envoy 将开始发送 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`。这是根据 :ref:`负载报告时间间隔 <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>` 定期进行的。
```
Comment (helight) [inline docs/root/intro/arch_overview/upstream/load_reporting_service.rst]:
```suggestion
这将启动一个与管理服务器的双向流。连接后，管理服务器可以给它想获取负载报告的节点发送一个 :ref:`LoadStatsResponse <envoy_v3_api_msg_service.load_stats.v3.LoadStatsResponse>` 。这个节点中的 Envoy 将开始发送 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`。这是根据 :ref:`负载上报时间间隔 <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>` 定期进行的。
```

**Review by helight — COMMENTED** [review]
一点建议，看看

File: docs/root/intro/arch_overview/upstream/load_reporting_service.rst (hunk 4 above)
Comment (webup) [reply docs/root/intro/arch_overview/upstream/load_reporting_service.rst]: 语序觉得现在的还可以就没改了

**Pull request merged**